CHANGELOG

```diff
@@ -1,6 +1,13 @@
 devel
 -----
 
+* added LDAP tests for fail-over or alternate LDAP servers. There are the
+  following test scenarios:
+ - two LDAP servers, each responsible for the same users. The second one
```

Comment:
I am unable to find this test

The below test scenarios we have. (different users, and different users, but one server is offline) but the two servers with same users seems to be not there, as the ldap2 host always uses a different responsible file, and uses a different postfix.

Reply:
removed entry, failure points will be used for failover tests. Will be added in this PR